packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/TimePickerContent.tsx

```diff
@@ -88,12 +88,10 @@ export const TimePickerContentWithScreenSize = (props: PropsWithScreenSize) => {
                 placeholder={t('time-picker.content.filter-placeholder', 'Search quick ranges')}
               />
             </div>
-            <CustomScrollbar>
```

Comment:
I think you have an unused import here.

Reply:
Super weird that my editor or pre-commit didn't complain

Reply:
Yeah, but if you have only a few items, it'll still show a scrollbar:
![Screenshot 2024-05-27 at 14 53 25](https://github.com/grafana/grafana/assets/8878045/c163bf3a-a533-4be4-b9d4-dc3ea1f6733d)

Reply:
Ooooh, I forgot that it was used there too. Thanks!